gradle/dependencies.gradle

```diff
@@ -21,8 +21,8 @@ ext.deps = [
         grammarKit: "com.github.hurricup:gradle-grammar-kit-plugin:${versions.idea}.1",
     ],
     kotlin: [
-        stdlib: "org.jetbrains.kotlin:kotlin-stdlib-jdk7:${versions.kotlin}",
         stdlibCommon: "org.jetbrains.kotlin:kotlin-stdlib-common:${versions.kotlin}",
+        stdlibJdk: "org.jetbrains.kotlin:kotlin-stdlib-jdk7:${versions.kotlin}",
```

Comment:
would be good to switch to the `kotlin.stdlib.jdk` style for these in a future pr

Reply:
Yeah almost did it here. Can do next.